src/components/SamlProviderConfiguration/index.test.jsx

```diff
@@ -33,7 +32,7 @@ describe('<SamlProviderConfiguration /> ', () => {
   it('get provider config (with no data)', () => {
     LmsApiService.getProviderConfig.mockResolvedValue(configResponse);
     LmsApiService.getProviderData.mockResolvedValue(invalidDataResponse);
-    const wrapper = mount(<SamlProviderConfiguration enterpriseId="testEnterpriseId" />);
+    const wrapper = mount(<SamlProviderConfigurationCore enterpriseId="testEnterpriseId" learnerPortalEnabled />);
```

Comment:
Since I now wrap the component default export in react-redux I would have to make quite a few changes to the tests and decided to instead test the underlying component here without worrying about how we fetch and feed the redux state value